src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/controller/v1_0/DisputeController.java

```diff
@@ -95,6 +95,20 @@ public ResponseEntity<Dispute> getDispute(@PathVariable Long id, Principal princ
 		}
 		return new ResponseEntity<Dispute>(disputeService.getDisputeById(id), HttpStatus.OK);
 	}
+	
+	/**
+	 * GET endpoint that retrieves the detail of a specific dispute without assigning it
+	 *
+	 * @param id
+	 * @param principal the logged-in user
+	 * @return {@link Dispute}
+	 */
+	@GetMapping("/dispute/noassign/{id}")
+	public ResponseEntity<Dispute> getDisputeNoAssign(@PathVariable Long id, Principal principal) {
+		logger.debug("GET /disputes/noassign/{} called", id);
+		return new ResponseEntity<Dispute>(disputeService.getDisputeById(id), HttpStatus.OK);
+	}
```

Comment:
I think instead of creating a new endpoint, you can get `isAssign` parameter or something into the existing one and update the logic accordingly.